package.json

```diff
@@ -4,6 +4,7 @@
   "scripts": {
     "prebuild": "./bin/playbook-changer.js",
     "build": "antora antora-playbook.yml $@",
+    "lint": "vale --config vale.ini docs/en/modules/admin/",
```

Comment:
@RCheesley I've added you as a reviewer. Let me know if the command that I'm adding in the task is similar to what you use locally or if you're using other flags/options.

Reply:
Hey there! I don't know much about npm lint, but I believe this would be limited only to the admin section of the docs which isn't really what you want, right?

You would normally use a GitHub Action to lint only the files that have been changed in the PR, and if you are working locally you either use the Vale plugin in your IDE or you run it at the CLI for the specific folder/file.

What is it that you're trying to achieve here? Maybe I missed that step.